### PR TITLE
Improve doc with response schema

### PR DESCRIPTION
**Pull request by maikmb**
Improve of `usecase` type with response schema for describe usecase response doc and validate response schema.

image

### CODE REVIEW

File: src/usecase.js [hunk 1]
```diff
@@ -65,13 +69,22 @@ class UseCase {
...
+        if (this._responseSchema) docStep.response = this._responseSchema
         return docStep
     }
 
     get auditTrail() {
         return this._mainStep.auditTrail
     }
 
+    get requestSchema() {
+        return this._requestSchema;
```
Comment (jhomarolo) [inline src/usecase.js:81]: This will break the linter checker

**Review by jhomarolo — CHANGES_REQUESTED** [review]
Hi Maik, your PR break the eslint checker. Can you fix it?

File: src/usecase.js (hunk 1 above)
Comment (maikmb) [reply src/usecase.js]: Hi @jhomarolo !
Fixed!

**Comment by maikmb** [comment]
> Hi Maik, your PR break the eslint checker. Can you fix it?



> Hi Maik, your PR break the eslint checker. Can you fix it?

Fixed!

**Comment by dalssoft** [comment]
It seems to be missing complex test scenarios for `response` like arrays `[String]`, etc

**Review by dalssoft — CHANGES_REQUESTED** [review]
Please, improve the tests with complex scenarios

**Comment by maikmb** [comment]
Hi @dalssoft !
I improved the tests with complex scenarios to response schema. Can you see please?

**Review by dalssoft — APPROVED** [review]

**Pull request merged**